classes/blocks/class-covers.php

```diff
@@ -381,16 +381,21 @@ private static function populate_posts_for_act_pages( &$fields ) {
 				if ( is_array( $wp_tags ) && $wp_tags ) {
 					foreach ( $wp_tags as $wp_tag ) {
 						$tags[] = [
-							'name' => $wp_tag->name,
+							'name' => html_entity_decode( $wp_tag->name ),
```

Comment:
Previously this data went to Twig, which simply included it as text in the HTML. However now we use it with React which would output the entity literally (e.g. `&amp;`)